src/cms-content/learn/products-landing.json

```diff
@@ -55,6 +55,12 @@
         }
       ]
     },
+    {
+      "productName": "FAQs",
```

Comment:
Oh boy, we are abusing this schema a bit. 🤣

Reply:
heh its gone through a couple revisions at this point

Reply:
but on my list of things to do